### DISCUSSION TITLE
New Install - CLUELESS

### DISCUSSION THREAD

**Discussion by Engineered4Fun** [o3de/o3de#2050] [Help Wanted] [answered]
o3de  --  I have gone through the whole tutorial and installed all the cmd prompts and I still can't get it to run!
Please help! I open the project in Visual Studio and it doesn't look anything like the game engine.  I'm new to development in general and would appreciate the help.

**Comment by OBWANDO** [comment, marked as answer]
Your fastest path is Discord in #sig-build or #general 
https://discord.gg/xNb2q4SJKJ

**Reply by Engineered4Fun** [reply]
Thanks, think I figured it out... after all the install you have to run the sln....  lots of steps involved

**Reply by Engineered4Fun** [reply]
![image](https://user-images.githubusercontent.com/87152384/125147833-83edb300-e0f3-11eb-8274-8b4a94973f8c.png)
Didn't work

**Reply by OBWANDO** [reply]
You should compile this in Release or in Profile, not Debug. Also if you are starting it from the visual studio UI, you need to look in the Solution explorer folders under Solution -> Code -> Editor -> Editor (right click on the one with the gears) and right click -> set as default project. 

OR just go into your root o3de folder and run this file in the path: /o3de/build/windows_vs2019/bin/profile/editor.exe

**Comment by Engineered4Fun** [comment]
Thank you so much, I was able to get it running now.
Thanks,
Dustin


________________________________

**Comment by Engineered4Fun** [comment]
Hello Royal,
I have hit another road block an I know that your knowledge can make this seem like an easy fix.
I am not able to drag the .cgf files onto the program.   It didn't know what program to open it with, so I chose o3de... apparently, that was not the correct choice.
Dustin

________________________________
